pmm/pmm2-release.groovy

```diff
@@ -337,14 +343,22 @@ ENDSSH
                             if [ -f /srv/repo-copy/pmm2-components/yum/release/7/RPMS/x86_64/repodata/repomd.xml.asc ]; then
                                 rm -f /srv/repo-copy/pmm2-components/yum/release/7/RPMS/x86_64/repodata/repomd.xml.asc
                             fi
+
+                            createrepo --update /srv/repo-copy/pmm2-components/yum/release/9/RPMS/x86_64/
+                            if [ -f /srv/repo-copy/pmm2-components/yum/release/9/RPMS/x86_64/repodata/repomd.xml.asc ]; then
+                                    rm -f /srv/repo-copy/pmm2-components/yum/release/9/RPMS/x86_64/repodata/repomd.xml.asc
+                            fi
+
                             export SIGN_PASSWORD=\${SIGN_PASSWORD}
                             gpg --detach-sign --armor --passphrase \${SIGN_PASSWORD} /srv/repo-copy/pmm2-components/yum/release/7/RPMS/x86_64/repodata/repomd.xml
+                            gpg --detach-sign --armor --passphrase \${SIGN_PASSWORD} /srv/repo-copy/pmm2-components/yum/release/9/RPMS/x86_64/repodata/repomd.xml
```

Comment:
Just a question: do we need an `if` statement here or is it idempotent?

Reply:
if you are talking about `if` in line 348 - yes we need it